src/devices/Media/VideoDevice/samples/V4l2.Samples.csproj

```diff
@@ -0,0 +1,12 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
```

Comment:
any specific reason for 3.0? other projects are fine with 2.1 (it's ok to be this way, just curious)

Reply:
My fault, I forgot to modify it.😂